### ISSUE TITLE
Tailor tile for a single style

### ISSUE THREAD

**Issue by wipfli** [loeli-style/loeli-style#15]
As a basic principle, I think we should tailor the tile for a single style. I am open to have some easy hooks for modifying the look in the front-end. But if people really want a different look, they should generate their own tiles.

This is a bit of a shift of paradigm from what vector renderers have done in the last 10 years. I think the paradigm was always to generate tiles once and then use and style them for as many things as possible. That restricts your freedom when generating tiles. I think today we can drop this restriction completely because with @msbarry's Planetiler, you can generate your own planet scale tileset in like 1 hour for 1 USD on the cloud provider of your choice...

I see Löli Style as a trivial map style which can get you started. And it will not be a very good map. If people want to customize, I suggest we encourage them to make their own tiles. I think that should be quite simple if we put one or two sentences in the READMEs.

**Comment by wipfli** [comment]
Related to #3

**Comment by wipfli** [comment]
Related to #5

**Comment by msbarry** [comment]
Maybe outside the scope of this project but I wonder if it would be possible to take a maplibre style as input and generate the tiles specifically for that? Or some layer on top of both that planetiler can run from and also generate a maplibre config.

**Comment by wipfli** [comment]
Yes something like that would be quite interesting. I think tile generation is part of the rendering process, so coupling frontend rendering and tile generation well together makes a lot of sense for me.

**Comment by ZeLonewolf** [comment]
> Maybe outside the scope of this project but I wonder if it would be possible to take a maplibre style as input and generate the tiles specifically for that? Or some layer on top of both that planetiler can run from and also generate a maplibre config. 

The pertinent with that is the maplibre style has no way of knowing how the underlying data mapped to tile attributes.

**Comment by maxammann** [comment]
Definitely an interesting idea/direction. Some unified configuration for tile generation and maplibre seems like a good idea to think about.
